docs/source/explore/get-started/install.rst

```diff
@@ -52,15 +52,61 @@ In addition, your network must have enough internet bandwidth to run Vircadia:
 .. note:: Running Vircadia on a high resolution display or in VR mode will need a much better graphics adapter, depending on your display's resolution and targeted frame rate.
 
 -----------------------------------
-Download Vircadia Installer
+Download Vircadia
 -----------------------------------
 
-You can download Vircadia through our website:
+* Windows |Windows|
 
-* `Download the Client Installer <https://vircadia.com/download-vircadia/#interface>`_
+    * `Windows 64-bit - Demeter v2020.3.0 Installer <https://cdn.vircadia.com/dist/release/Vircadia-Alpha-2020.3.1-Demeter.exe>`__ |DownloadWI|
+    * `Windows 64-bit Launcher (Experimental) <https://cdn.vircadia.com/dist/launcher/VircadiaLauncherSetup.exe>`__ |DownloadWL|
 
+* macOS |MacOS|
+
+    * `macOS Intel - Demeter v2020.3.0 Installer <https://cdn.vircadia.com/dist/release-experimental/Vircadia-Alpha-Interface-2020.3.1-Demeter.dmg>`__ |DownloadM|
+
+* Linux |Ubuntu|
+
+    * `EU Mirror: Linux amd64 - Demeter v2020.3.1 AppImage <https://appimage.moto9000.moe/release/Vircadia-x86_64_v2020.3.1_release.AppImage>`__ |DownloadLEU|
+..
+    * `US Mirror: Linux amd64 - Demeter v2020.3.0 AppImage <https://cdn.vircadia.com/dist/release/Vircadia-Alpha-2020.3.0-Demeter_build2.AppImage>`__ |DownloadLUS|
 
 If you intend to use Vircadia in VR mode with an HMD, ensure that SteamVR or Oculus Runtime is also installed on your system before launching Vircadia.
+VR mode is not supported on macOS. The Oculus Runtime is not supported on Linux.
+
+
+.. |Windows| image:: _images/remixicon/windows-fill.svg
+              :class: inline
+
+.. |MacOS| image:: _images/remixicon/finder-fill.svg
+              :class: inline
+
+.. |Ubuntu| image:: _images/remixicon/ubuntu-fill.svg
+              :class: inline
+
+.. |DownloadWI| image:: _images/remixicon/download-2-fill.svg
+              :class: inline
+              :width: 20
+              :target: https://cdn.vircadia.com/dist/release/Vircadia-Alpha-2020.3.1-Demeter.exe
+
+.. |DownloadWL| image:: _images/remixicon/download-2-fill.svg
+              :class: inline
+              :width: 20
+              :target: https://cdn.vircadia.com/dist/launcher/VircadiaLauncherSetup.exe
+
+.. |DownloadM| image:: _images/remixicon/download-2-fill.svg
+              :class: inline
+              :width: 20
+              :target: https://cdn.vircadia.com/dist/release-experimental/Vircadia-Alpha-Interface-2020.3.1-Demeter.dmg
+
+.. |DownloadLEU| image:: _images/remixicon/download-2-fill.svg
+              :class: inline
+              :width: 20
+              :target: https://appimage.moto9000.moe/release/Vircadia-x86_64_v2020.3.1_release.AppImage
+
+.. |DownloadLUS| image:: _images/remixicon/download-2-fill.svg
+              :class: inline
+              :width: 20
+              :target: https://cdn.vircadia.com/dist/release/Vircadia-Alpha-2020.3.0-Demeter_build2.AppImage
```

Comment:
```suggestion
              :target: https://cdn.vircadia.com/dist/release-experimental/Vircadia-Alpha-2020.3.1-Demeter.AppImage
```

Reply:
I don't see why the AppImages would be considered experimental again

Reply:
Yeah, I had forgotten that I'd upgraded that to "release" since it's been performing quite well as of late. :) Fixed.